src/FormData.ts

```
@@ -2,8 +2,8 @@ import path from 'node:path';
 
 import _FormData from 'form-data';
 
-// eslint-disable-next-line
-const NON_ASCII_RE = /[^\x00-\x7F]/i;
+// eslint-disable-next-line no-control-regex
+const NON_ASCII_RE = /[^\u0000-\u007F]/i;
```

Biome (2.4.6)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)